core/field_dropdown.js

```diff
@@ -441,15 +447,6 @@ Blockly.FieldDropdown.prototype.updateColour = function() {
  * @private
  */
 Blockly.FieldDropdown.prototype.render_ = function() {
-  var child;
```

Comment:
This goes away because you're creating the image and arrow elements in initView and keeping them around, right?

Reply:
Correct.